packages/cadl-ranch-specs/http/server/version/query/main.tsp

```diff
@@ -0,0 +1,37 @@
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+@doc("Illustrates server with multiple versions and the version is in query.")
+@scenarioService("/server/version/query", Options)
+@server(
+  "{endpoint}",
+  "Testserver endpoint",
+  {
+    @doc("Need to be set as 'http://localhost:3000' in client.")
+    endpoint: url,
+  }
+)
+namespace Server.Version.Query;
+
+enum Versions {
+  v1_1: "v1.1",
+  v1_2: "v1.2",
+}
```

Comment:
I assume this case already covered it?
https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/resiliency/srv-driven/main.tsp#L40-L47